app/routes/data.js

```diff
@@ -1,7 +1,17 @@
 import Route from '@ember/routing/route';
 import layerGroups from '../layer-groups';
 
+/**
+ * The data route is responsible for statically importing layer groups
```

Comment:
what does 'displaying a layer groups information' mean? Are these for displaying the selected geo when viewing a profile? or for displaying the map used to create selections? or both?

Reply:
@julialucyhogan I don't really know, the template has markup for looping over the layer-groups which are used in the _map_ when during the geo selection process: https://factfinder-staging.planninglabs.nyc/data#12.25/40.724/-73.9868.

Reply:
I think this should become a separate issue